lib/puppet/face/certificate.rb

```diff
@@ -113,27 +113,27 @@
       $ puppet certificate sign somenode.puppetlabs.lan --ca-location remote
     EOT
 
-    option("--[no-]allow-dns-alt-names") do
+    option("--[no-]allow-subject-alt-names") do
```

Comment:
I think we will need the same aliasing and deprecating here as we have in the other command.

Reply:
I just added the deprecation warning, but I'm not sure if/how we need to alias allow-dns-alt-names? It looks like we're  using ``options[:allow_subject_alt_names]`` which should have already been aliased if they entered --allow-dns-alt-names, right?